app/javascript/toast-ui.scss

```diff
@@ -0,0 +1,66 @@
+@import "@toast-ui/editor/dist/toastui-editor.css";
+@import "@toast-ui/editor/dist/theme/toastui-editor-dark.css";
+
+.markdown-editor {
+  * {
+    box-sizing: content-box;
+  }
+}
+
+// /*------------------------------------*\
+//     $ToastUI overrides
+// \*------------------------------------*/
+
+// Overrides for the preview section to match real output styles
+.toastui-editor-contents {
+  font-size: 14px;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    margin: 0.5em 0 0.5em 0;
+    padding: 0;
+    border-bottom: 0;
+    font-weight: normal;
+  }
+
+  blockquote {
+    border-left: 4px solid var(--bs-secondary-bg);
+    p {
+      color: var(--bs-secondary-text-emphasis);
+    }
+  }
+
+  ul > li::before {
+    content: none;
+  }
+
+  ul {
+    list-style-type: disc;
+  }
+
+  ul ul {
+    list-style-type: circle;
+  }
+
+  ol > li::before {
+    color: black;
+  }
+
+  code {
+    color: var(--bs-code-color);
+  }
```

Comment:
(This is probably not the correct location, but it's related to code highlighting.)

I noticed that code blocks are not correctly display in the Markdown preview. Below is an example:

<img width="911" alt="Bildschirmfoto 2024-04-14 um 21 22 47" src="https://github.com/openHPI/codeocean/assets/7300329/ba3787b7-2831-4fec-a1e1-0ffd7841fe4c">

It seems like this part is being applied, too, making this weird background:

https://github.com/openHPI/codeocean/blob/944b4551940fddd90556badc59d6fd3f7e296979/app/assets/stylesheets/exercises.css.scss#L1-L5